extensions/example-extension/renderer.tsx

```diff
@@ -9,17 +9,39 @@ export default class ExampleExtension extends LensRendererExtension {
       title: "Example Extension",
       components: {
         Page: () => <ExamplePage extension={this}/>,
```

Comment:
I also think it'll be more intuitive to (also) provide param values here as a `props` object that can be passed into the component that gets rendered when the page is activated:
```javascript
components: {
  Page: (props) => <ExamplePage extension={this} exampleId={props.params.exampleId} />,
}
// or just
components: {
  Page: ({ params }) => <ExamplePage extension={this} params={params} />,
}
// or whatever the developer wants to do
```
This can be in addition to `Navigation.getPageParam()` if we want to keep both ways. IMO, this props way is more intuitive than `Navigation.getPageParam()`.

Reply:
right, and no reason to add `@observer` in that case